src/Management/src/EndpointBase/Health/HealthConverter.cs

```diff
@@ -26,17 +26,10 @@ public override void Write(Utf8JsonWriter writer, HealthEndpointResponse value,
                     writer.WriteString("description", health.Description);
                 }
 
-                if (health.Details != null && health.Details.Count > 0)
+                foreach (var detail in health.Details)
```

Comment:
This looks like a reversal of #518, maybe slipped through during a rebase?

Reply:
I was at git hell level 1 and tried to be 'clever' and ended up in git hell level 2. 
I've made some changes that I hope will resolve things.